### ISSUE TITLE
Enhance select hide below field

### ISSUE THREAD

**Issue by hamed-alshukri** [evertiro/historical-redux2#33]
It would be much better if you extend the select hide below field to hide multiple field at once and not only the one below the select field. 

The idea is to specify the fields class name you want to hide in the "allow" arg. The idea can be applied for radio and checkbox, multi and single, fields.

**Comment by psaikali** [comment]
Commited here > https://github.com/psaikali/Redux-Framework/commit/39a6a3a01380022b6cc125f2896864b4676e6441

```
        array(
            'id' => 'general_pretitle',
            'type' => 'checkbox_hide_below',
            'title' => __('Hide next 4 fields', Redux_TEXT_DOMAIN),
            'desc' => __('Hide next 4 fields', Redux_TEXT_DOMAIN),
            'next_to_hide' => 3
        ),
```

Will hide/show the next 3 fields.

**Comment by evertiro** [comment]
Resolved with commit 7dd901620a4dc24acaf5870e52c0a89dae61acc0

**Comment by abdullah-dev11** [comment]
this resolved only with checkbox , not with select
